docs/debugging-testing/testing/testing_greentea.md

```diff
@@ -27,10 +27,10 @@ int main() {
 In addition to being placed under a `TESTS` directory, test sources must exist under two other directories: a test group directory and a test case directory. The following is an example of this structure:
 
 ```
-myproject/TESTS/test_group/test_case_1
+mbed-os/mylibrary/tests/TESTS/test_group/test_case_1
```

Comment:
I would not change the example with `myproject`; it is still valid and it was clearer.